src/server/components/serializers/DelmalSerializer.tsx

```diff
@@ -46,12 +65,18 @@ const DelmalSerializer = (props: IDelmalSerializerProps) => {
               FlettefeltSerializer({
                 sanityProps: props,
                 flettefelter,
-                dokumentApiNavn: apiNavn,
+                dokumentApiNavn: delmalApiNavn,
+              }),
+            utbetalinger: (_: any) =>
+              UtbetalingerSerializer({
+                maalform: maalform,
+                utbetalingerPerMndEøs: (dokumentData as IDokumentDataMedUtbetalingerEøs)
```

Comment:
Skulle gjerne hatt en [typeguard](https://www.typescriptlang.org/docs/handbook/advanced-types.html) her i stedet for å caste. Også kan vi kaste feil om det ikke er EØSdata

Reply:
Litt usikker på hvordan du ser for deg dette, men har pushet opp et forslag.